### PR TITLE
[MTDSA-3286] Update the GitHub Issues templates

### PR DESCRIPTION
**Pull request by james-work-account**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -7,7 +7,26 @@ assignees: ''
...
+- Issues integrating with the API Platform (e.g. setting up an application/access)
+- Code examples in a given programming language
+- Business-related questions or processes pre MTD
+- Scheduled maintenance windows and days
+- Onboarding (e.g. registering, obtaining keys)
+- Performance metrics
+
+For any of the above, you **must** contact the software developer support team directly at SDSTeam@hmrc.gov.uk.  The SDS team do not monitor this repository.
+
+This repository is for bug reports relating to the VAT API, fill in the sections below to submit your bug report:
```
Comment (jeremystone) [inline .github/ISSUE_TEMPLATE/bug_report.md]: ...VAT API only. 
<new para>
Fill in the sections below....

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -7,6 +7,27 @@ assignees: ''
...
+- Issues integrating with the API Platform (e.g. setting up an application/access)
+- Code examples in a given programming language
+- Business-related questions or processes pre MTD
+- Scheduled maintenance windows and days
+- Onboarding (e.g. registering)
+- Performance metrics
+
+For any of the above, you **must** contact the software developer support team directly at SDSTeam@hmrc.gov.uk.  The SDS team do not monitor this repository.
+
+This repository is for feature requests relating to the VAT API only, fill in the sections below to submit your feature request:
```
Comment (jeremystone) [inline .github/ISSUE_TEMPLATE/feature_request.md]: ...VAT API only.

Fill in the sections below....

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -7,7 +7,28 @@ assignees: ''
...
-**This bug is related to the**
+# Bug report
+
+You must read the information below before you submit a bug report. 
+
+The MTD API team cannot help if your bug report is about:
+
+- Fraud Prevention Headers
+- Issues relating to OAuth2
+- Subscribing or registering to use the API Platform, MTD enrolment, de-enrolment, registration for VAT, Beta services etc.
```
Comment (ma3574) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Should there be a comma after “Beta Services”

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -7,7 +7,28 @@ assignees: ''
...
+- Issues integrating with the API Platform (e.g. setting up an application/access)
+- Code examples in a given programming language
+- Business-related questions or processes pre MTD
+- Scheduled maintenance windows and days
+- Onboarding (e.g. registering, obtaining keys)
+- Performance metrics
+
+For any of the above, you **must** contact the software developer support team directly at SDSTeam@hmrc.gov.uk.  The SDS team do not monitor this repository.
```
Comment (ma3574) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Software Developer Support Team (capitalised as that’s their name)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -7,7 +7,28 @@ assignees: ''
...
+
+This repository is for bug reports relating to the VAT API only.
```
Comment (ma3574) [inline .github/ISSUE_TEMPLATE/bug_report.md]: VAT API code in this repository only

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (james-work-account) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Googled it. Yes.

**Pull request merged**